tests/test_js_prefs.py

```diff
@@ -23,6 +23,17 @@ def test_pref_innocuous_branch():
     pref("extensions.foo-bar.baz", true);
     """, path="defaults/preferences/prefs.js").failed()
 
+def test_unicode_pref():
+    """Tests that non-ASCII preferences do not cause errors."""
+
+    assert not _do_test_raw("""
+    pref("extensions.foo-bar.\u263a", true);
+    """, path="defaults/preferences/prefs.js").failed()
+
+    assert not _do_test_raw("""
+    pref("extensions.foo-bar.\\u263a", true);
+    """, path="defaults/preferences/prefs.js").failed()
+
```

Comment:
Hm. The first test just needs to be a `u` string. As it is now, the `\u263a` gets passed through literally. With that change, the first test fails prior to this fix. With #294, they both fail prior to this fix.